canvas_sdk/v1/data/allergy_intolerance.py

```diff
@@ -20,7 +20,6 @@ class Meta:
     dbid = models.BigIntegerField(primary_key=True)
     created = models.DateTimeField()
     modified = models.DateTimeField()
-    editors = ArrayField(models.IntegerField())
```

Comment:
Haha... yeahh.... I'm not sure how to handle that just yet either. I believe supporting an ArrayField would make my idea of using a sqlite database for local development more difficult.